packages/cli/src/commands/setup/auth/templates/auth0.auth.ts.template

```diff
@@ -1,4 +1,4 @@
-import { parseJWT } from '@redwoodjs/api'
+import { parseJWT, Decoded } from '@redwoodjs/api'
```

Comment:
Same, maybe `import type` instead so it get's removed from JS files

Reply:
Thanks for commenting on this. Turns out `parseJWT` isn't actually used for auth0... So I removed that and made this into a type import only 🙂